lib/portlayer/exec/handle.go

```diff
@@ -147,6 +147,19 @@ func (h *Handle) Commit(ctx context.Context, sess *session.Session, waitTime *in
 	// make sure there is a spec
 	h.SetSpec(nil)
 	cfg := make(map[string]string)
+
+	// Set timestamps based on target state
+	switch *h.State {
```

Comment:
No - I have it specifically before the commit so it will be added to the VM spec -- then during the reconfigure ops for start / stop it will make it to the VMX